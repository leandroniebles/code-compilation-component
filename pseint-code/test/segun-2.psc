Proceso sin_titulo
	1: Escribir 1+2;
	Segun a Hacer
		1+"B": escribir "A";
	FinSegun
FinProceso
