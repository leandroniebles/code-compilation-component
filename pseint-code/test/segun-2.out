=== Line 2: SynError 241
=== Line 4: SynError 295
